data/hai/hai missions.txt

```diff
@@ -1742,3 +1742,446 @@ mission "Nanachi Meeting"
+	shipyard "Hai Advanced"
+		"Pond Strider"
+		"Flea"
+	shipyard "Hai Basics"
+		"Flea"
+	outfitter "Hai Basic"
+		"Railgun Slug"
+	outfitter "Hai Intermediate"
+		"Railgun"
+		"Railgun Slug"
+		"Railgun Slug Rack"
+		"Sand Cell"
+		`"Baellie" Atomic Engines`
+	outfitter "Hai Advanced"
+		"Railgun"
+		"Railgun Slug"
+		"Railgun Slug Rack"
+		"Sand Cell"
+		`"Baellie" Atomic Engines`
+	fleet "Large Hai"
+		add variant 1
+			"Pond Strider" 4
+			"Flea" 32
+			"Geocoris"
+		add variant 1
+			"Centipede (Pulse)" 3
+			"Centipede" 2
+			"Pond Strider" 3
+			"Flea" 24
+		add variant 1
+			"Pond Strider (Railgun)" 3
+			"Flea (Pulse)" 24
+			"Aphid" 10
+		add variant 2
+			"Pond Strider (Pulse)" 3
+			"Flea" 24
+			"Lightning Bug (Pulse)" 2
+		add variant 2
+			"Pond Strider (Ion)" 3
+			"Flea" 24
+			"Lightning Bug" 2
+		add variant 1
+			"Pond Strider (Ion)" 3
+			"Flea" 24
+			"Lightning Bug (Pulse)"
+			"Aphid" 4
+		add variant 1
+			"Pond Strider (Railgun)" 3
+			"Flea" 24
+			"Lightning Bug"
+			"Aphid (Anti-Missile)" 4
+		add variant 2
+			"Pond Strider" 3
+			"Flea (Pulse)" 24
+			"Lightning Bug (Pulse)"
+			"Water Bug"
+			"Water Bug (Pulse)"
+		add variant 3
+			"Pond Strider" 3
+			"Flea (Pulse)" 12
+			"Flea" 12
+			"Water Bug"
+			"Water Bug (Pulse)"
+		add variant 2
+			"Pond Strider" 3
+			"Flea" 24
+			"Lightning Bug" 2
+			"Water Bug"
+		add variant 3
+			"Pond Strider" 3
+			"Flea" 24
+			"Lightning Bug (Pulse)" 2
+		add variant 2
+			"Pond Strider (Pulse)" 3
+			"Flea" 24
+			"Shield Beetle"
+		add variant 2
+			"Shield Beetle (Pulse)"
+			"Pond Strider (Railgun)" 3
+			"Flea" 24
+		add variant 1
+			"Pond Strider (Ion)" 3
+			"Flea" 24
+			"Shield Beetle"
+			"Lightning Bug (Pulse)" 2
+		add variant 1
+			"Shield Beetle (Pulse)"
+			"Pond Strider (Tracker)" 3
+			"Flea" 24
+			"Lightning Bug" 2
+		add variant 1
+			"Pond Strider (Tracker)" 3
+			"Flea" 24
+			"Centipede" 3
+		add variant 2
+			"Centipede"
+			"Shield Beetle (Pulse)" 2
+			"Pond Strider (Railgun)" 3
+			"Flea" 24
+		add variant 1
+			"Geocoris"
+			"Shield Beetle (Pulse)"
+			"Pond Strider (Tracker)" 3
+			"Flea" 24
+		add variant 1
+			"Geocoris"
+			"Pond Strider (Pulse)" 3
+			"Flea" 24
+			"Grasshopper"
+			"Grasshopper (Tracker)"
+		add variant 1
+			"Geocoris"
+			"Pond Strider (Pulse)" 3
+			"Flea" 24
+			"Grasshopper"
+			"Grasshopper (Tracker)"
+		add variant 1
+			"Geocoris (Tracker)"
+			"Aphid (Anti-Missile)" 2
+			"Aphid"
+			"Pond Strider" 3
+			"Flea" 24
+
+mission "Strider 4"
+	landing
+	source "Hai-home"
+	to offer
+		or
+			has "Strider 3 (Remnant): done"
+			has "Strider 3 (Coalition): done"
+	on offer
+		event "pond strider mass production" 30
+		conversation
+			``
+				decline
+
+mission "Strider notification"
+	invisible
+	landing
+	source
+		government "Hai"
+	to offer
+		has "event: pond strider mass production"
+	on offer
+		log `For assisting the Hai establish an alien embassy, they've made newly developed warships and outfits available.`
```

Comment:
While this log entry is good for the Coalition route, it doesn't match the Remnant route. Having separate log entries would be good, but alternatively slightly modifying it as follows would also work.

```suggestion
		log `For assisting the Hai conduct negotiations with an alien faction, they have made newly developed warships and outfits available.`
```